src/systemFlake.nix

```diff
@@ -19,12 +19,21 @@
     extraArgs = sharedExtraArgs;
   }
 
-, packagesBuilder ? null
-, defaultPackageBuilder ? null
-, appsBuilder ? null
-, defaultAppBuilder ? null
-, devShellBuilder ? null
-, checksBuilder ? null
+, outputsBuilder ? channels: {
```

Comment:
Naming: I really like `outputsBuilder`. I don't think we should use `exports`. Any better ideas?